runtime/src/bank.rs

```diff
@@ -3629,6 +3642,10 @@ impl Bank {
             &solana_sdk::sysvar::id(),
         )
         .unwrap();
+
+        info!(
+            "create epoch rewards partition data account {} {address} {epoch_rewards_partition_data:?}", self.slot
+        );
```

Comment:
It would actually be nice to output this in the `force_partition_rewards_in_first_block_of_epoch()` case. Can you rework `create_epoch_rewards_partition_data_account()` to take `force_partition_rewards_in_first_block_of_epoch: bool` and wrap L3549?

Reply:
Fixed.